tests/mail/tests.py

```diff
@@ -1448,6 +1448,19 @@ def mock_send(self, s):
         finally:
             SMTP.send = send
 
+    def test_send_messages_after_open_failed(self):
+        """Test that no messages are sent after open() silently failed."""
```

Comment:
State the expected behavior rather than  including a "Test that" prefix. All tests do that. For example, "send_messages() shouldn't try to send messages if open() failed to connect."